src/pages/experiments/[experimentId]/plots-and-tables/components/LabelsDesign.jsx

```diff
@@ -7,19 +7,15 @@ import {
 
 const LabelsDesign = (props) => {
   const { config, onUpdate } = props;
-  // const [labelsEnabled, setlabelsEnabled] = useState(true);
-  const onUpdateThrottled = useRef(_.throttle((obj) => onUpdate(obj), 10));
 
-  const onChange = (e) => {
-    onUpdate({ labelsEnabled: e.target.value, labelShow: e.target.value ? 1 : 0 });
-  };
+  const onUpdateThrottled = useRef(_.throttle((obj) => onUpdate(obj), 50));
```

Comment:
does this actually work? Does it throttle?

Reply:
Yes, it works in staging

Reply:
that's awesome, we can look at the code here for the throttling bug ticket we have in the sprint